src/slices/sidebar/discussion/comments-list/Node.js

```diff
@@ -36,6 +39,7 @@ export default class Node extends PureComponent {
 
     state = {
         editing: false,
+        enteredEditingMode: false,
```

Comment:
Isn't `enteredEditingMode` a misleading variable name for its current purpose? The code leads me to conclude whenever an edit operation is finished, whether successfully or not, its value changes to `true`. As such, shouldn't it be named something like `exitingEditingMode`? What's the relationship with `editing` state variable?